subscriptions/tasks.py

```diff
@@ -12,10 +12,33 @@
 from .models import Subscription
 from contentstore.models import Message
 from scheduler.client import SchedulerApiClient
+from go_http.metrics import MetricsApiClient
 
 logger = get_task_logger(__name__)
 
 
+class FireMetrics(Task):
+
+    """ Fires a metric using the MetricsApiClient
+    """
+
+    def set_metric_client(self):
+        return MetricsApiClient(
+            auth_token=settings.METRICS_AUTH_TOKEN,
+            api_url=settings.METRICS_URL,
+            session=None)
```

Comment:
Not if you do `def set_metric_client(self, session=None):` and make this `session=session` ... which you should 😄